src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/DocumentController.kt

```diff
@@ -75,6 +76,26 @@ class DocumentController(
     )
   }
 
+  @GetMapping("/recalls/{recallId}/documents")
+  fun getRecallDocumentsByCategory(
+    @PathVariable("recallId") recallId: RecallId,
+    @RequestParam("category") category: DocumentCategory
+  ): ResponseEntity<List<Api.RecallDocument>> {
+    return ResponseEntity.ok(
+      documentService.getAllDocumentsByCategory(recallId, category).map { document ->
+        Api.RecallDocument(
+          document.id(),
+          document.category,
+          document.fileName,
+          document.version,
+          document.details,
+          document.createdDateTime,
+          userDetailsService.get(document.createdByUserId()).fullName()
```

Comment:
Future: I can see us wanting the userId on this response as well as the name.

Reply:
Personally, feels like we should avoid exposing uuids to clients. but 🤷